packages/jest/test/__snapshots__/preact.test.js.snap

```diff
@@ -1,21 +1,56 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`jest-emotion with preact handles elements with no props 1`] = `"<div />"`;
+exports[`jest-emotion with preact handles elements with no props 1`] = `
+"Object {
+  \\"__\\": null,
+  \\"__b\\": 0,
+  \\"__c\\": null,
+  \\"__d\\": undefined,
+  \\"__e\\": null,
+  \\"__h\\": null,
+  \\"__k\\": null,
+  \\"__v\\": 3,
+  \\"constructor\\": undefined,
+  \\"key\\": undefined,
+  \\"props\\": Object {},
+  \\"ref\\": undefined,
+  \\"type\\": \\"div\\",
+}"
+`;
 
 exports[`jest-emotion with preact replaces class names and inserts styles into preact test component snapshots 1`] = `
-".emotion-0 {
-  color: red;
-}
-
-.emotion-1 {
-  width: 100%;
-}
-
-<div
-  class=\\"emotion-0\\"
->
-  <svg
-    class=\\"emotion-1\\"
-  />
-</div>"
+"Object {
```

Comment:
I guess those tests show that it's actually still used/tested with Preact but the changed output here indicated that it's perhaps no longer working that well with Preact 10. Maybe we can still remove those tests and Preact as a dependency? If Preact users are using Emotion they might not use `@emotion/jest` - cause it looks like this combination is not a good match at the moment

Reply:
The snapshots look pretty useless as is. I did some research and it looks like `preact-render-to-json` has been superseded by `preact-render-to-string`. I switched these tests to `preact-render-to-string` and the snapshots looked good, but didn't contain the CSS from Emotion since there is no place to pass in the Emotion serializer thing.

I think it's very unlikely anyone is using the Emotion Jest serializer with Preact 10, so I think we can just remove these tests. What do you think?

Reply:
Ye, I would drop those tests